src/Workspaces/Remote/ServiceHub/UserOperationBooster.cs

```diff
@@ -31,10 +31,16 @@ public static UserOperationBooster Boost()
             {
                 s_count++;
 
-                if (s_count == 1)
+                try {
+                    if (s_count == 1)
+                    {
+                        // boost to normal priority
+                        Process.GetCurrentProcess().PriorityClass = ProcessPriorityClass.Normal;
+                    }
+                } catch (Exception)
```

Comment:
I don't think it's good idea to use a general exception. Specify the exact exception type being thrown.